### PR TITLE
Add github actions

### PR DESCRIPTION
**Pull request by deniskoronchik**
* [ ] Read PR [documentation](http://ostis-dev.github.io/sc-machine/dev/pr/)
* [ ] Update changelog
* [x] Update documentation

Resolves: #361

### CODE REVIEW

**Review by KovalM — APPROVED** [review]

File: docs/dev/pr.md [hunk 1]
```diff
@@ -12,17 +12,22 @@ Message text should start from an upper case letter. If commit doesn't fix or im
...
+  * `[cmake]` - changes in cmake build system
+  * `[codegen]` - changes in code generator
+  * `[core]` - changes in `sc-storage` code
+  * `[doc]` or `[docs]` - update documentation
+  * `[git]` - changes in git configuration
+  * `[python]` - changes in python modules
+  * `[sc-kpm]` - changes in `sc-kpm` code
+  * `[sc-memory]` - changes in `sc-memory` code
+  * `[sc-server]` - changes in `sc-server` code
+  * `[sc-network]` - changes in `sc-network` code
```
Comment (MikhailSadovsky) [inline docs/dev/pr.md:26]: Please swap `[sc-network]` with `[sc-server]` lines to keep alphabetical order

File: docs/dev/pr.md [hunk 2]
```diff
@@ -12,17 +12,22 @@ Message text should start from an upper case letter. If commit doesn't fix or im
...
+  * `[scs]` - changes in `scs` support
+  * `[test]` or `[tests]` - changes in tests
+  * `[thirdparty]` - updated thirdparty libraries
+  * `[web]` - changes in web interface
```
Comment (MikhailSadovsky) [inline docs/dev/pr.md:31]: We probably need to add `scripts` and `travis` tags to this list

Comment (deniskoronchik) [reply docs/dev/pr.md:31]: Maybe we will use common `ci` instead of `travis`?

Comment (MikhailSadovsky) [reply docs/dev/pr.md:33]: Agree, lets use `ci`

**Review by MikhailSadovsky — APPROVED** [review]

**Pull request merged**